specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -469,6 +470,10 @@ packages. Merging changes into `main` requires that these pipelines pass
 successfully.
 
 ##### e2e test image
+
+Used for running end-to-end tests for the `data-pipelines` package. Exposes all
+`data-pipelines` features in order to run tests. Not user-facing.
+
```

Comment:
I see the changes, you may update the stacked PR as well.
I don't understand how "exposes all data-pipelines features" differs from quickstart-vdk image, I suppose it also exposes these features?

May explain what Frontend artifacts come built-in in e2e test image, and what of those is installed after, and why?

Reply:
I'm not sure at the moment what we'll hide and what we'll end up hiding and exposing for the UI image. They're technically the same right now. Off the top of my head, the UI image will have:
- csp authentication disabled
- the Explore page disabled
- download keytab button disabled

But it might not be limited to those, so I'd rather not add description for stuff that is not currently implemented. That should be in the scope of another ticket.

Reply:
You can also point out here that the e2e image contains a browser and webpack. 
Neither of these are in the production image.

Reply:
I thought some more about this. When we do e2e testing, we're testing the application as a black box. The testing environment is irrelevant to the application itself. E2e testing and shipping an artefact are two different things. E2e testing requires scaffolding, for example. We see that with bundling Chrome in the docker image. There may be feature flags passed, etc. 

The point is that the app artefact itself doesn't really change, even though configs, environments, and packaging change. So I don't think we should explicitly do e2e testing on the prod Docker image. The artefact itself doesn't change between building the e2e tests image and the production image.